core/src/main/java/org/dst/core/DstConcurrentHashMap.java

```diff
@@ -0,0 +1,15 @@
+package org.dst.core;
+
+public interface DstConcurrentHashMap<K, V> {
+
+  void put(K key, V value);
+
+  V remove(K key);
```

Comment:
1. The return value of `remove` should be a `boolean`, which indicates whether we do a remove actually.

2. Could you add some comment of THESE methods?

Reply:
Fine.  I will add comments to these methods